doc/source/whatsnew/v0.19.0.txt

```diff
@@ -3,15 +3,16 @@
 v0.19.0 (August ??, 2016)
 -------------------------
 
-This is a major release from 0.18.2 and includes a small number of API changes, several new features,
```

Comment:
@jorisvandenbossche slight changes need here (which are in master). done here i think is fine.